docs/VnVPlan/VnVPlan.tex

```diff
@@ -69,141 +70,158 @@ \section{Symbols, Abbreviations, and Acronyms}
 
 \subsection{Relevant Documentation}
 
-\wss{Reference relevant documentation.  This will definitely include your SRS
-  and your other project documents (design documents, like MG, MIS, etc).  You
-  can include these even before they are written, since by the time the project
-  is done, they will be written.}
-
-\citet{SRS}
-
-\wss{Don't just list the other documents.  You should explain why they are relevant and 
-how they relate to your VnV efforts.}
+% \wss{Reference relevant documentation.  This will definitely include your SRS
+%   and your other project documents (design documents, like MG, MIS, etc).  You
+%   can include these even before they are written, since by the time the project
+%   is done, they will be written.}
+
+Listed below are the relevant documents referred to in this VnV Plan:
+
+\begin{itemize}
+  \item \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/SRS/SRS.pdf}{Software Requirements Specification} \citet{GrocerySRS}\\
+  The SRS contains the primary functional and non-functional requirements that are being used as the basis for many of the tests in this document.
+  \item \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/HazardAnalysis/HazardAnalysis.pdf}{Hazard Analysis} \citet{GroceryHA}\\
+  The HA contains a number of safety requirements that were identified to help mitigate risk that were also considered for testing in this document.
+  \item \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/Design/SoftArchitecture/MG.pdf}{Module Guide}\\
+  The MG references the different modules that make up the general overall application in which the VnV Plan tests.
+  \item \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/Design/SoftDetailedDes/MIS.pdf}{Module Interface Specification}\\
+  The MIS contains detailed information on each of the application modules which form the basis for the unit tests.
+  \item \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/DevelopmentPlan/DevelopmentPlan.pdf}{Development Plan} \citet{GroceryDP}\\
+  The DP describes a roadmap for the project including timelines, communication plans, and technical details.
+\end{itemize}
 
 \section{Plan}
 
-\wss{Introduce this section.   You can provide a roadmap of the sections to
-  come.}
+The following will outline a series of plans and processes that will be used for the verification and validation of the software solution and documentation the Grocery Spending Tracker project. The sections are as follows:
+\begin{itemize}
+    \item Verification and Validation Team: the team members their involvement in the verification and validation efforts 
+    \item SRS Verification Plan: the approaches to be used for SRS verification
+    \item Design Verification Plan: plans for design verification
+    \item Verification and Validation Plan Verification Plan: plans for VnV verification
```

Comment:
is this typo? cause the sentence is confusing...

Reply:
no lol
![image](https://github.com/r-yeh/grocery-spending-tracker/assets/32990185/9f5fb8c0-e53f-4b76-9875-e4969d1ac53d)

Reply:
WTF LOLL